views.py

```diff
@@ -769,7 +779,8 @@ def filter(course_id=None):
 
 
 @app.route('/launch', methods=['POST'])
-def lti_tool():  # pragma: no cover
+@lti(error=error, request='initial', role='staff', app=app)
+def lti_tool(lti=lti):  # pragma: no cover
```

Comment:
Since we've removed all the ims_lti_py cruft, this function can be reasonably tested. Recommend removing `# pragma: no cover`

Reply:
removed.